index.html

```diff
@@ -2,29 +2,31 @@
 
 <html>
   <head>
+    <style>
+      @import url("https://fonts.googleapis.com/css2?family=Oswald&display=swap");
```

Comment:
You can just add this to your style sheet that is already linked.

Reply:
like add it to the actual CSS file? not my index.html file?

Reply:
got it! I just put it on line 1 , of the CSS file, is that the best place for it?